src/components/ApplicationSpecificMenu/ApplicationSpecificMenu.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from "react";
+import { cn } from "../../core/utils";
+
+interface MenuItem {
+  name: string;
+  icon?: string;
```

Comment:
should be a react component React.ReactNode instead of raw svg string.
```suggestion
  icon?: React.ReactNode;
```